app/helpers/admin/claims_helper.rb

```diff
@@ -50,5 +51,22 @@ def display_school(school)
       ].join(" ").html_safe
       sanitize(html, tags: %w[span a], attributes: %w[href class])
     end
+
+    def checking_deadline_warning(claim)
+      weekdays_until_checking_deadline = weekdays_between(Date.today, claim.checking_deadline_date)
+      return if weekdays_until_checking_deadline > 10
+
+      checking_deadline_warning_class = weekdays_until_checking_deadline < 0 ? "tag--alert" : "tag--warning"
+      content_tag(:strong, pluralize(weekdays_until_checking_deadline, "day"), class: "govuk-tag #{checking_deadline_warning_class}")
+    end
+
+    private
+
+    def weekdays_between(first_date, second_date)
+      dates_in_order = [first_date, second_date].sort
+      weekdays = (dates_in_order.first..dates_in_order.last).select { |date| [0, 6].exclude?(date.wday) }.count
```

Comment:
Reckon we can simplify this line with [`Date.#on_weekday?`](https://api.rubyonrails.org/v5.1/classes/DateAndTime/Calculations.html#method-i-on_weekday-3F):

```suggestion
      weekdays = (dates_in_order.first..dates_in_order.last).count { |date| date.on_weekday? }
```